src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  width: 200px;
+  height: 408px;
```

Comment:
do not hardcode height - it should depends on content

Reply:
not fixed https://github.com/mate-academy/layout_product-cards/pull/3668/files#r1349665534